### ISSUE TITLE
How to get the top-down map and the trajectory?

### ISSUE THREAD

**Issue by lizixiaoboluo** [cshizhe/VLN-DUET#21]
Hi, thanks for your great work.
Can you share how to get the top-down map and the corresponding trajectory?
thank you very much

**Comment by ZhenLiu-xjtu** [comment]
Hi，Have you solved this problem？

**Comment by ZhouZineng** [comment]
Hi, have you solved this problem? I would like to know the answer too.

**Comment by ZhenLiu-xjtu** [comment]
No, I have no clue about this. Are you also using Duet as a baseline? If so, we can discuss it

-----Original Messages-----
From:ZhouZineng ***@***.***>
Send time:Saturday, 01/04/2025 21:59:46
To: cshizhe/VLN-DUET ***@***.***>
Cc: ZhenLiu-xjtu ***@***.***>, Comment ***@***.***>
Subject: Re: [cshizhe/VLN-DUET] How to get the top-down map and the trajectory? (Issue #21)






Hi, have you solved this problem? I would like to know the answer too.

**Comment by cshizhe** [comment]
Do you mean the visualization? We modified the visualization codes [here](https://github.com/niessner/Matterport/tree/master/code/gaps/apps/mpview). But this was done long time ago, we do not keep the codes now.

**Comment by ZhouZineng** [comment]
Hello, I am referring to this kind of top-view image in the paper. Figure 5. Predicted trajectories of DUET and the state-of-the-art HAMT
![topview](https://github.com/user-attachments/assets/00ac4294-1591-4f34-8f19-078cf9c807e4)


> Do you mean the visualization? We modified the visualization codes [here](https://github.com/niessner/Matterport/tree/master/code/gaps/apps/mpview). But this was done long time ago, we do not keep the codes now.

**Comment by cshizhe** [comment]
Yes, this is implemented based on the [matterport visualization code](https://github.com/niessner/Matterport/tree/master/code/gaps/apps/mpview).

**Comment by AIerOK** [comment]
> Hello, I am referring to this kind of top-view image in the paper. Figure 5. Predicted trajectories of DUET and the state-of-the-art HAMT ![topview](https://github.com/user-attachments/assets/00ac4294-1591-4f34-8f19-078cf9c807e4)
> 
> > Do you mean the visualization? We modified the visualization codes [here](https://github.com/niessner/Matterport/tree/master/code/gaps/apps/mpview). But this was done long time ago, we do not keep the codes now.

Hello, I also encountered this issue. Have you resolved it?
